servicetalk-concurrent-reactivestreams/src/test/java/io/servicetalk/concurrent/reactivestreams/tck/SingleAmbWithTckTest.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright © 2020 Apple Inc. and the ServiceTalk project authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.servicetalk.concurrent.reactivestreams.tck;
+
+import io.servicetalk.concurrent.api.Single;
+
+import org.testng.annotations.Test;
+
+import static io.servicetalk.concurrent.api.Single.never;
+
+@Test
+public class SingleAmbWithTckTest extends AbstractSingleOperatorTckTest<Integer> {
```

Comment:
what about tests for the `Completable` variants?

Reply:
Ya I did not add it as it was simply delegating to `Single#amb()` but lemme add the basic tests for it.